src/api/spec/models/user_spec.rb

```diff
@@ -278,10 +278,26 @@
         expect(user.email).to eq('some@email.com')
```

Comment:
What is the point of this shared example? Seems unnecessary to test basic active record behaviour.